go.mod

```diff
@@ -2,17 +2,15 @@ module github.com/urfave/cli-altsrc/v3
 
 go 1.23.2
 
-require github.com/stretchr/testify v1.9.0
+require (
+	github.com/BurntSushi/toml v1.4.0
+	github.com/stretchr/testify v1.9.0
+	github.com/urfave/cli/v3 v3.0.0-beta1
+	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
- `json/json_value_source.go` uses `gopkg.in/yaml.v2`
- `yaml/yaml_value_source.go` uses `gopkg.in/yaml.v3`

Should we use only `gopkg.in/yaml.v3` for consistency?

Reply:
Yes. Good observation.